### ISSUE TITLE
Issue Tracker config

### ISSUE THREAD

**Issue by leo-bianchi** [gogs/go-gogs-client#98]
Hello, i'm trying to set a issue tracker config based on wiki. On "Edit Issue Tracker" session

I'm using the following command:

$ curl -u "myuser" -d "issue.json" -H "Content-Type: application/json" -X PATCH http://myurl/api/v1/repos/myuser/myrepo/issue-tracker

The command it's not working. 
Am I doing it wrong?

**Comment by Unknwon** [comment]
>The command it's not working.

Be specific.

**Comment by leo-bianchi** [comment]
Hello. 
I'm following the wiki instructions...
https://github.com/gogs/go-gogs-client/wiki/Repositories#edit-issue-tracker
So...
I create the json file, with the same parameters...
```json
{
  "enable_issues": true,
  "enable_external_tracker": true,
  "external_tracker_url": "https://github.com/gogs/gogs/issues",
  "tracker_url_format": "https://github.com/gogs/{repo}/issues/{index}"
}
```
And I have been trying commands in Ubuntu 18 to create repos... to sync... etc
and i'm using curl to all this things.
To create a new repo I run
```bash
$ curl -d "@file.json" -H "Content-Type: application/json" -X POST myurl/api/v1/user/repos?token=mytoken.
```
That's working pretty well. 
and I've trying to edit issue tracker configs via command line with curl too, and it's not working.
i'm using Ubuntu 18. I've installed Gogs today. i'ts running on localhost:3000. 
How can i be more specific?

**Comment by Unknwon** [comment]
First of all, no Gogs version is mentioned. Secondly, “not working” is as vague as said nothing. What is expect and what actually you got?

PS: this API is just added few days ago so no release has it unless you build from source code from master branch.

**Comment by leo-bianchi** [comment]
Gogs Version: 0.11.66.0916. I installed from binary in https://gogs.io/docs/installation/install_from_binary
I expected the command just change the issue tracker configs, like set a external url issue tracking, enable or disable issue tracking on my repository... 
I did'nt receive a error message... 
when i ran command... I receive nothing.
Thanks for 'PS' we can conclude that it's not working because I'm using a version of Gogs that not implements this feature. Agree?
Very grateful
